reactor-core/src/test/java/reactor/core/scheduler/DelegateServiceSchedulerTest.java

```diff
@@ -49,6 +50,31 @@ protected boolean shouldCheckDisposeTask() {
 		return false;
 	}
 
+	@Override
+	protected boolean shouldCheckSupportRestart() {
+		return false;
+	}
+
+	@Test
+	public void startAndDecorationImplicit() {
+		AtomicInteger decorationCount = new AtomicInteger();
+		Schedulers.setExecutorServiceDecorator("startAndDecorationImplicit", (s, srv) -> {
+			decorationCount.incrementAndGet();
+			return srv;
+		});
+		final Scheduler scheduler = new DelegateServiceScheduler("startAndDecorationImplicitExecutorService", Executors.newSingleThreadExecutor());
+
+		try {
+			assertThat(decorationCount).as("before schedule").hasValue(0);
+			scheduler.schedule(() -> {});
+			assertThat(decorationCount).as("after schedule").hasValue(1);
+		}
+		finally {
+			scheduler.dispose();
+			Schedulers.removeExecutorServiceDecorator("startAndDecorationImplicit");
```

Comment:
nit: will (should?) also be removed by the global test listener that runs the cleanups

Reply:
this indeed predates the global test listener. there is currently no way to remove ESDecorators without providing a name, though.